src/notation/internal/notationactioncontroller.cpp

```diff
@@ -1329,7 +1329,7 @@ void NotationActionController::openBreaksDialog()
 
 void NotationActionController::openScoreProperties()
 {
-    interactive()->open("musescore://notation/properties");
+    interactive()->open("musescore://project/scoreproperties");
```

Comment:
URI should be:

musescore://project/properties

Also, the action "edit-info" should be moved to the project module. And I would suggest renaming it to "project-properties"